wallets/index.js

```diff
@@ -48,17 +48,17 @@ export function useWallet (name) {
   const _isConfigured = isConfigured({ ...wallet, config })
 
   const enablePayments = useCallback(() => {
-    enableWalletLocally(name, me)
+    saveConfig(config => ({ ...config, enabled: true }), { skipTests: true })
```

Comment:
My understanding was that enable/disablePayments are supposed to disable or enable a payment method for the local machine, while the state in the vault should change only by unchecking `enabled` in the wallet settings

Reply:
Ah, I see. No, `enabled` (and `priority`) should always be part included in the wallet configuration on any device.